modules/cloud-conf-shiftstack-reference.adoc

```diff
@@ -0,0 +1,245 @@
+//  Module included in the following assemblies:
+
+//  * installing/installing_openstack/installing-openstack-cloud-config-reference.adoc
+
+:_content-type: REFERENCE
+[id="cloud-conf-shiftstack-reference_{context}"]
+= OpenStack cloud provider options 
+
+The cloud provider configuration, typically stored as a file named `cloud.conf` controls how OpenShift Container Platform interacts with Red Hat OpenStack Platform (RHOSP).
+
+You can create a valid `cloud.conf` file by specifying the following options in it.
+
+[id="ccm-config-global-options"]
+== Global options
+
+The following options are used for {rh-openstack} CCM authentication with the {rh-openstack} Identity service, also known as Keystone. They are similiar to the global options that you can set by using the `openstack` CLI.
+
+|===
+| Option | Description
+
+| `application-credential-id`
+| The ID of an application credential to authenticate with. An `application-credential-secret` must be set along with this parameter.
+
+| `application-credential-name`
+| The name of an application credential to authenticate with. If `application-credential-id` is not set, the user name and domain must be set.
+
+| `application-credential-secret`
+| The secret of an application credential to authenticate with.
+
+| `auth-url`
+| The {rh-openstack} Identity service URL. For example, `http://128.110.154.166/identity`.
+
+| `ca-file`
+| Optional. The CA certificate bundle file for communication with the {rh-openstack} Identity service. If you use the HTTPS protocol with The Identity service URL, this option is required.
+
+| `cert-file`
+| Optional. The client certificate path to use for client TLS authentication.
+
+| `key-file`
+| Optional. The client private key path to use for client TLS authentication.
```

Comment:
These two will probably not work. Context: https://github.com/kubernetes/legacy-cloud-providers/blob/release-1.23/openstack/openstack.go#L158-L173